src/commands/edge/channels/invites/create.ts

```diff
@@ -41,6 +42,7 @@ const builder = (yargs: Argv): Argv<CommandArgs> =>
 				'create an invite from prompted input for the specified channel',
 			],
 		])
+		.epilog(buildEpilog({ command }))
```

Comment:
This is an example of a command that doesn't strictly need an update. Since it's a leaf command and doesn't use any notes or have any API docs, this effectively does nothing. (The `buildEpilog` function in this case returns an empty string which yargs is happy to ignore, or maybe it appends it, either having the same effect. 😄)

I'm kind of leaning towards including this in even these cases but I could be convinced otherwise. The main reason to add it would be that if we don't and a command gets sub-commands added, we will have to remember to update the parent command with an epilog.